tests/test_vector2_angle.py

```diff
@@ -10,7 +10,7 @@
     (Vector2(1, 1), Vector2(-1, 0), 135),
     (Vector2(0, 1), Vector2(0, -1), 180),
     (Vector2(-1, -1), Vector2(1, 0), 135),
-    (Vector2(-1, -1), Vector2(-1, 0), 45),
+    (Vector2(-1, -1), Vector2(-1, 0), -45),
```

Comment:
Why are there unrelated changed in the angle tests?

Reply:
Because passing tests.

Reply:
Except that the tests are passing in `master`, so there is something wrong going on.

Reply:
You're going to make my life difficult, I know it.

Reply:
*OK.*

This is actually a bug mypy found.

The two tests in this file had the same name, so only the latter one was being run.

This exposed problems in the earlier test because it wasn't running before.

This can be moved to its own PR if y'all would really rather.